src/actions/sources/newSources.js

```diff
@@ -171,11 +179,10 @@ export function newSources(sources: Source[]) {
       dispatch(checkPendingBreakpoints(source.id));
     }
 
-    await Promise.all(
-      filteredSources.map(source => dispatch(loadSourceMap(source.id)))
-    );
+    await dispatch(loadSourceMaps(filteredSources));
```

Comment:
We should load source maps for only the generated sources, how about we incorporate that change here as well?

Reply:
Thoughts on this @jasonLaster or can we merge?

Reply:
done. good idea!